test/operations-conditions-2.js

```diff
@@ -0,0 +1,19 @@
+/*2. Переводчик
+Напишите функцию helloWorld которая: Принимает 1 аргумент, код языка (например "en", "es") верните "Hello, World" для данного языка(хотя бы три языка).
+ По умолчанию возвращает русский вариант.
+ */
+
+function helloWorld(lang) {
+  switch (lang) {
+    case "en":
+      return "Hello";
+    case "es":
+      return "Hola";
+    default:
+      return "Привет";
+  }
+}
+console.log(helloWorld());
```

Comment:
in real projects - such log messaged considered as debugging info and it should not be present in the final version